apps/web/cypress/e2e/register.cy.ts

```diff
@@ -0,0 +1,193 @@
+import { Web } from "@litespace/utils/routes";
+
+describe("registration", () => {
+  beforeEach(() => {
+    cy.visit(Web.Register.replace(":role", "student"));
```

Comment:
This is not how we populate urls. This approach is unsafe and prone to errors. To manage urls we use a specific class call `RoutesManager`. Checkout `apps/web/src/lib/routes.ts`.  You will need a do a similar think for cypress. To do this, you need to create file at `cypress/lib/routes.ts`

```ts
import { RoutesManager } from "@litespace/utils/routes";
export const router = new RoutesManager('local');
```

Then later in `/cypress/e2e/register.cy.ts` import it 

```ts
import {router} from '@cy/lib/routes.ts'
```

Look for examples in the codebase to understand how to use it.